checks/evaluation/pinned_dependencies_test.go

```diff
@@ -205,6 +207,34 @@ func Test_PinningDependencies(t *testing.T) {
 				NumberOfDebug: 0,
 			},
 		},
+		{
+			name: "various wanrings and info",
```

Comment:
the purpose of the test isn't immediately clear to me from the name or what its expecting

Reply:
Updated the name to reflect better message